testkit/tests/system_api.rs

```diff
@@ -40,3 +40,12 @@ fn test_user_agent_info() {
     let expected = user_agent::get();
     assert_eq!(info, expected);
 }
+
+#[test]
+#[should_panic(expected = "Cannot send data")]
```

Comment:
Checking against the message seems overly fragile. Also, I don't quite understand the test purpose; wouldn't it pass for some non-existing endpoint (e.g., `v2/shutdown`)?